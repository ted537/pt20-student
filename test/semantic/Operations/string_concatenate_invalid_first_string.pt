using input;
var str = 5 | "World";
