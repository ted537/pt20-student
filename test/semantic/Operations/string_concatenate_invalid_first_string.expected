   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 1
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
   .tInitialValue
     .tLiteralInteger
     oEmitValue
     % value emitted 5
      .tSkipString
      oEmitNullAddress
      % value emitted -32767
      .tStringData
      oEmitValue
      % value emitted 5
      oEmitString
      % value emitted 87
      % value emitted 111
      % value emitted 114
      % value emitted 108
      % value emitted 100
      % value emitted 0
     oEmitNullAddress
     % value emitted -32767
     .tLiteralAddress
     .tFetchChar
     oEmitValue
     % value emitted 18
     .tConcatenate
       #eTypeMismatch
   .tInitEnd
   .tLiteralAddress
    .tStoreChar
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
